### DISCUSSION TITLE
What does the Getorganelle output file mean?

### DISCUSSION THREAD

**Discussion by Mrwangrqqq** [Kinggerm/GetOrganelle#252] [Q&A]
Hello teacher, I would like to ask what the output file after assembling the chloroplast genome means? Thank you teacher！
![屏幕截图 2023-03-30 230202](https://user-images.githubusercontent.com/129409292/228879555-a6e7585c-cde8-4e45-9023-6d5baaedefc2.png)

**Comment by JianjunJin** [comment]
See detailed illustration for the results through [Key results section at the homepage](https://github.com/Kinggerm/GetOrganelle#starting-from-reads) and through [Ouput section at the example](https://github.com/Kinggerm/GetOrganelle/wiki/Example-1#output-files).
